assets/tests/components/map.test.tsx

```diff
@@ -137,6 +137,32 @@ describe("map", () => {
     await userEvent.click(screen.getByText(runIdToLabel(vehicle.runId!)))
     expect(onClick).not.toHaveBeenCalled()
   })
+
+  test("renders street view link if in maps test group", async () => {
+    const shape = {
+      id: "shape",
+      points: [
+        { lat: 0, lon: 0 },
+        { lat: 0, lon: 0 },
+      ],
+      stops: [
+        {
+          id: "stop",
+          name: "stop",
+          lat: 0,
+          lon: 0,
+        },
+      ],
+    }
+    render(<Map vehicles={[]} shapes={[shape]} />)
+
```

Comment:
Hmmm I see what you mean about the difficulty testing the popups -- _maybe_ something like `await userEvent.click(container.querySelector("m-vehicle-map__stop")!)` would be possible for simulating a click on a stop, but querying by class name is not considered best practice in react-testing-library. 

This may be something that would eventually be able to cover with something like [cypress tests](https://app.asana.com/0/1152340551558956/1203152611716214/f).

Reply:
I was gonna see if I could render `LeafletShape` directly to make it easier to avoid class names as you said, but I think I'll use that to get something working at first.

Something I noticed, the stop buttons don't show up in the accessibility tree, do you know any background as to if we could make the stops `role=button` or something?

Reply:
It may be a limitation of using the existing leaflet `CircleMarker` which has limited props - perhaps we could have greater control over customization if we used a regular `Marker` with full control over the underlying html